### PR TITLE
[UI] Rename Dashboard to Telemetry

### PR DESCRIPTION
**Pull request by JessicaDJ0807**
Since the original app calls Telemetry as Dashboard, to eliminate ambiguity, the related module and folder are renamed to Telemetry.

BUG: b/297124938

### CODE REVIEW

File: diagnostics-app/src/app/app.module.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { environment } from '../environments/environment';
 import { AppRoutingModule } from './app-routing.module';
```
Comment (BaiiYuan) [inline diagnostics-app/src/app/app.module.ts:16]: Please remove the extra newline in line 20 and sort these import header in alphabetical order.

File: diagnostics-app/src/app/telemetry/telemetry.module.ts [hunk 2]
```diff
@@ -25,4 +25,4 @@ import { CpuComponent } from './cards/cpu/cpu.component';
   imports: [CommonModule, SharedModule],
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
 })
-export class DashboardModule {}
+export class TelemetryModule {}
```
Comment (BaiiYuan) [inline diagnostics-app/src/app/telemetry/telemetry.module.ts:28]: Please update line 6.

**Review by BaiiYuan — COMMENTED** [review]
Finished 1st round review.

File: diagnostics-app/src/app/app.module.ts (hunk 1 above)
Comment (JessicaDJ0807) [reply diagnostics-app/src/app/app.module.ts]: Done.

File: diagnostics-app/src/app/telemetry/telemetry.module.ts (hunk 2 above)
Comment (JessicaDJ0807) [reply diagnostics-app/src/app/telemetry/telemetry.module.ts:28]: Done. I also updated the fileoverview parts in "cpu.component.ts", "memory.component.ts", and "storage.component.ts".

**Review by ycheng627 — APPROVED** [review]
LGTM

**Pull request merged**